Anna_Golovenko/js/scripts.js

```diff
@@ -23,24 +23,45 @@ function initCalculator() {
 
     event.preventDefault();
 
+    if (isNaN(result) || (result === undefined)) {
```

Comment:
In this case You can write all in one line:
```javascript
if (isNaN(result) || result === undefined) return;
```
And `===` operator has a higher priority than `||`, and You can omit `( ... )` brackets.